### ISSUE TITLE
Startup error after update to 4.1.0

### ISSUE THREAD

**Issue by carpac** [t1m0thyj/WinDynamicDesktop#207]
Hi,
on my Windows 10 laptop, WinDynamicDesktop 4.0.0 was installed. When I updated to 4.1.0 today, the program starts with an error message and dies. In the log file I found the following entry (partially in German as my Windows installation is German):

```
[2020-05-20 15:08:52.567] System.FormatException: Die Zeichenfolge wurde nicht als gültiges DateTime erkannt.
   bei System.DateTimeParse.Parse(String s, DateTimeFormatInfo dtfi, DateTimeStyles styles)
   bei WinDynamicDesktop.UpdateChecker.TryCheckAuto(Boolean forceIfEnabled)
   bei WinDynamicDesktop.UpdateChecker.Initialize()
   bei WinDynamicDesktop.AppContext..ctor(String[] args)
   bei WinDynamicDesktop.Program.Main(String[] args)

```
In the end I had to delete the settings.conf file. Then WinDynamicDesktop starts successfully, of course asking me for some configuration details. It seems to me as if the German date format is not recognized correctly in 4.1.0.

**Comment by juicypop** [comment]
That's strange. I'm also using DDW on a german Windows 10 but haven't encountered this issue yet. Does it return after reboot?

**Comment by carpac** [comment]
Yes, it was the same after reboot and it occurred on two computers. Both had v4.0.0 installed before.

I think the date format is the reason for this error. In the settings.conf of v4.0.0 the date strings looked like "20.05.2020 16:45:54" which is the German date format. After deleting settings.conf and performing a new configuration, the date format is now like "05/20/2020 17:11:32".

Maybe the root of the error lies in an earlier version of WinDynamicDesktop as on both computers I started using WinDynamicDesktop many months before and since then just updated. Until v4.0.0 the date format did not seem to matter...

**Comment by sungryeol-bigpi** [comment]
Same issue here: it's written in Korean but basically the same meaning. It says it has an issue while parsing the target string into DateTime. Windows 10 Home Korean version.
```
[2020-05-21 14:55:53.964] System.FormatException: 문자열을 유효한 DateTime으로 인식할 수 없습니다. 11 인덱스에서 시작하는 알 수 없는 단어가 있습니다.
   위치: System.DateTimeParse.Parse(String s, DateTimeFormatInfo dtfi, DateTimeStyles styles)
   위치: WinDynamicDesktop.WallpaperShuffler.MaybeShuffleWallpaper()
   위치: WinDynamicDesktop.WallpaperChangeScheduler.RunScheduler(Boolean forceImageUpdate)
   위치: WinDynamicDesktop.AppContext..ctor(String[] args)
   위치: WinDynamicDesktop.Program.Main(String[] args)

[2020-05-21 14:56:43.043] System.FormatException: 문자열을 유효한 DateTime으로 인식할 수 없습니다. 11 인덱스에서 시작하는 알 수 없는 단어가 있습니다.
   위치: System.DateTimeParse.Parse(String s, DateTimeFormatInfo dtfi, DateTimeStyles styles)
   위치: WinDynamicDesktop.WallpaperShuffler.MaybeShuffleWallpaper()
   위치: WinDynamicDesktop.WallpaperChangeScheduler.RunScheduler(Boolean forceImageUpdate)
   위치: WinDynamicDesktop.AppContext..ctor(String[] args)
   위치: WinDynamicDesktop.Program.Main(String[] args)
```

**Comment by t1m0thyj** [comment]
This should be fixed in 4.1.1 (was a pretty serious issue so warranted a patch release).
